quickfilter.go

```diff
@@ -179,7 +179,12 @@ func (qf QuickFilter) UnionOf(qf1, qf2 QuickFilter) QuickFilter {
 	qf.len = 0
 	for i := range qf.bits {
 		qf.bits[i] = qf1.bits[i] | qf2.bits[i]
-		qf.len += bits.OnesCount(qf.bits[i])
+
+		if i == len(qf.bits) - 1 {
```

Comment:
Instead of performing this check for each element, we could do it once at the end and do the loop for all elements except last, same for intersection.